pkgs/unified_analytics/USAGE_GUIDE.md

```diff
@@ -7,7 +7,7 @@ To get started using this package, import at the entrypoint dart file and
 initialize with the required parameters
 
 ```dart
-import 'dash_analytics';
+import 'unified_analytics';
```

Comment:
=> `import 'package:unified_analytics/unified_analytics.dart'` ?

Reply:
Ah good catch, not having the analysis server in md files got me there :)